test/jdk/sun/tools/jstatd/JstatGCUtilParser.java

```diff
@@ -43,9 +43,9 @@ public enum GcStatisticsType {
     }
 
     public enum GcStatistics {
-        S0(GcStatisticsType.PERCENTAGE),
-        S1(GcStatisticsType.PERCENTAGE),
-        E(GcStatisticsType.PERCENTAGE),
+        S0(GcStatisticsType.PERCENTAGE_OR_DASH),
+        S1(GcStatisticsType.PERCENTAGE_OR_DASH),
+        E(GcStatisticsType.PERCENTAGE_OR_DASH),
```

Comment:
I don't see jstat tests in the zgc problem list. Is this fixing known test failures?

Reply:
Yes, ZGC reports dash for eden and survivor spaces.

Reply:
So shouldn't we have ZGC test failures then?

Reply:
No, test passes with ZGC.

Reply:
Before my changes test just silently ignored any GC setting and always use G1.

Reply:
Ah, ok. That makes sense now.